src/tutorials/konnector/2fa.md

```diff
@@ -0,0 +1,62 @@
+You have successfully developped your connector but you realize that some accounts need two factor
+authentication (2FA) to work.
+
+Here is how you can add this to your connector
+
+## 2FA detection
+
+Every target website being different, you will have to develop it yourself. Some websites return a
+specific url, some others display a special html content.
```

Comment:
Even if this is different for every target, I think adding a link to a real example can be a good thing.

Reply:
There are some links in the end of the document, some more here will be good too, yes.